pkg/gateway/vald/handler/grpc/handler.go

```diff
@@ -61,6 +62,8 @@ func New(opts ...Option) vald.ValdServer {
 }
 
 func (s *server) Exists(ctx context.Context, meta *payload.Object_ID) (*payload.Object_ID, error) {
+	ctx, _, end := trace.StartSpan(ctx, "vald/gateway-vald.Exists")
+	defer end()
 	uuid, err := s.metadata.GetUUID(ctx, meta.GetId())
 	if err != nil {
```

Comment:
I think we should set error status for each span
```suggestion
	if err != nil {
		span.SetStatus(trace.Status{
			Code:    trace.StatusCodeUnknown,
			Message: err.Error(),
		})
```